src/db/prisma/schema.prisma

```diff
@@ -0,0 +1,72 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Job {
+  id              String          @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  data            Json            @default("{}")
+  status          OperationStatus @default(PENDING)
+  percentage      Int             @default(0)
+  creation_time   DateTime?       @default(now())
+  update_time     DateTime?       @updatedAt
```

Comment:
should it be nullable?

Reply:
added a comment to remove it after CRUD implementation (so I will test it with prisma-client)